.gitattributes

```diff
@@ -0,0 +1,4 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text eol=lf
```

Comment:
According to the [gitattributes man page](https://git-scm.com/docs/gitattributes), `eol=lf` enables end-of-line normalization **without any content checks**. I am afraid that it may ruin non-text files which might be added in future. Actually I already have binary files in my custom branches.